docs/DEVELOPMENT.md

```
@@ -113,15 +113,15 @@ When adding a new container definition in some pod yaml, make sure to tag the `i
 field with `  ## OSC_VERSION`, e.g.
 
 ```
-image: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel9:1.12.0  ## OSC_VERSION
+image: registry.redhat.io/openshift-sandboxed-containers/osc-monitor-rhel9:1.12.1  ## OSC_VERSION
 ```
 
 Do the same when adding new `RELATED_IMAGE` entries in the environment of the controller
 in `config/manager/manager.yaml`, e.g.
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)